### PR TITLE
Popup window

### PR DESCRIPTION
**Pull request by m05e5**
-we designed the popup window
-we implemented the JS objects save the card information before opening the popup window

### CODE REVIEW

**Comment by NJ-2020** [comment]
Dear Reviewer,

I and Moses have created a popup window with the following features:

When the user clicks (or taps) the button to check project details, the popup with details about the project appears.
When the user clicks (or taps) the close (X) button, the popup disappears.

The window can both pop up on the mobile screen as well as the desktop screen.

Thanks

File: index.html [hunk 1]
```diff
@@ -32,7 +32,7 @@
     <!-- nav start -->
     <nav>
       <div class="logo">
-        <img src="IMG/my-logo.png" alt="">
+        <img src="IMG/my-logo.png" alt="" />
```
Comment (descholar-ceo) [inline index.html]: - [ ] [Optional] Wherever you have images, it's not good to leave `alt=' '` as empty, you should give them explicit descriptive value, for the sake of you site's accessibility!

File: index.html [hunk 2]
```diff
@@ -119,100 +119,292 @@ <h3 class="work-name">Tonic</h3>
...
+              <li class="info-modal">CANOPY</li>
+              <li><i class="info-sep fa fa-circle"></i></li>
+              <li class="info-modal">Back End Dev &nbsp;</li>
+                <li><i class="info-sep fa fa-circle"></i></li>
+                <li class="info-description info-modal">2015</li>
+              </ul>
+              
+              <div class="modal-body">
+                <img
+                src="img/work-img/work-preview1.png"
```
Comment (descholar-ceo) [inline index.html]: - [ ] On the popup window, the project image is broken, I think it would be very nice if you fixed it

File: index.html [hunk 3]
```diff
@@ -119,100 +119,292 @@ <h3 class="work-name">Tonic</h3>
...
+                <a href="https://github.com" class="button-modal">See Live <img src="IMG/Icon.png" alt="" /></a>
+                <a href="https://github.com" class="button-modal">See Source <img src="IMG/gitHubIcon.svg" alt="" /></a>
+              </div>
+
+              <div class="modal-footer">
+                <a href="#">Previous Project</a>
+                <a href="#">Next Project</a>
+              </div>
+            </div>
+          </div>
```
Comment (descholar-ceo) [inline index.html]: - [ ] We should make sure that the popup follow the given design template from figma

File: index.html [hunk 4]
```diff
@@ -119,100 +119,292 @@ <h3 class="work-name">Tonic</h3>
...
+            <button type="button" data-modal-target="#modal1" class="work-btn">
+              See Project
+            </button>
           </div>
-        </li>
+          <div class="modal" id="modal1">
+            <div class="modal-header">
+              <div class="title">Tonic</div>
+
+              <button type="button" data-close-button class="close-button">&times;</button>
```
Comment (descholar-ceo) [inline index.html]: - [ ] When a user clicks the `X` button on the popup window, the window should disappear

File: script.js [hunk 5]
```diff
@@ -1,4 +1,4 @@
-/* eslint-disable linebreak-style */
```
Comment (descholar-ceo) [inline script.js]: - [ ] I think you shouldn't disable linters rules, you should fix them instead

File: script.js [hunk 6]
```diff
@@ -8,4 +8,41 @@ menuToggle.addEventListener('click', () => {
...
+
+function openModal(modal) {
+  if (modal == null) return;
+  modal.classList.add('active');
+  overlay.classList.add('active');
+}
+
+function closeModal(modal) {
+  if (modal == null) return;
+  modal.classList.remove('active');
```
Comment (descholar-ceo) [inline script.js]: - [ ] The JavaScript code implements an array of objects to store JavaScript objects containing the information for each project.
- [ ] Each object contains at least the following information about the project: name, description, featuring image, technologies, link to live version, and link to the project source code. 
  e.g:
```javascript
const projects = [{
    name: "Submarine game",
    description: "A Phaser3 plataformer game built as part of the Microverse JavaScript program.",
    image: "link_to_image.jpg",
    technologies: ["Phaser3", "JavaScript", "Jest"],
    link: "link_to_live_version.com",
}]
```

- [ ] The projects section is created dynamically with the information from the projects array (all the HTML of the section is created when the page loads)
 i.e:
```javascript
window.onload = () => {
  // The DOM elements creation or functions that create them are called here.
};
```

**Review by descholar-ceo — CHANGES_REQUESTED** [review]
Hi @NJ-2020 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by NJ-2020** [comment]
Dear Reviewer, 

I have updated my portfolio based on the above feedback.
Please note that the live demo is not working properly, and I am not sure how to repair it.
Please download it for review, and the branch name is popup-window.  
I include a screenshot for clarification.

Many thanks
Novell

File: index.html (hunk 2 above)
Comment (DeliceLydia) [reply index.html]: Well addressed this!

File: index.html (hunk 3 above)
Comment (DeliceLydia) [reply index.html]: - [ ] The popup window does not follow the figma template please refer to that check the [Template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) out.

File: index.html (hunk 4 above)
Comment (DeliceLydia) [reply index.html]: Well resolved this!

File: script.js (hunk 5 above)
Comment (DeliceLydia) [reply script.js]: Addressed this.

File: script.js (hunk 6 above)
Comment (DeliceLydia) [reply script.js]: Well done you worked on this!

**Review by DeliceLydia — CHANGES_REQUESTED** [review]
Hi @NJ-2020, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments I added under the previous review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._
Consider:
- Check out your images links it might be the reason why images are not showing on the live demo
- I recommend that you follow the design on your contact section
template
contact

yours
fk

- Check your README it contains other names that are not yours like github handle please be careful on that


### Cheers and Happy coding!👏👏👏

Feel free to leave a question, or a comment on the PR thread if something is not clear 100%. 

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by NJ-2020** [comment]
Dear Reviewer, 

I have properly addressed all feedback.
In case the live demo does give you the proper outcome, please download it.

Many thanks
novell

**Review by Shaher-11 — APPROVED** [review]
#### Hello @m05e5 ,

# Congratulations! 🎉 
#### Your project is approved!  There is nothing else to say other than... it's time to merge it :muscle: :muscle::muscle:


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Cheers and Happy coding!👏👏👏

**Pull request merged**